sql/core/src/test/scala/org/apache/spark/sql/TPCDSBase.scala

```diff
@@ -79,6 +79,17 @@ trait TPCDSBase extends SharedSparkSession with TPCDSSchema {
        """.stripMargin)
   }
 
+  def createTables(): Unit = {
```

Comment:
where else do we call this method?

Reply:
> where else do we call this method?

Since PlanStabilitySuite extends TPCDSBase and the create table in `beforeAll`, here we split `createTables`
then for TPCH suite we can only just create our needed table.